multigeiger/multigeiger.ino

```diff
@@ -130,7 +130,7 @@ void display(unsigned long current_ms, unsigned long current_counts, unsigned lo
     if (afterStartTime && ((current_ms - boot_timestamp) >= afterStartTime)) {
       afterStartTime = 0;
       DisplayGMC(0, 0, 0,
-                 (showDisplay && switches.display_on), wifi_connected);
+                 (showDisplay && switches.display_on), wifi_connected, false); // last false in preparation for future BLE connectivity
```

Comment:
maybe add ble_connected or ble_active as variable to main loop (and put the comment there), and use the variable here?